RefrigeratorSaraFarkash/RefrigeratorSaraFarkash/CItem.cs

```diff
@@ -0,0 +1,124 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace RefrigeratorSaraFarkash
+{
+    internal class CItem:IComparable<CItem>
+    {
+
+        private static int countObj = 0;
+        private int id;
```

Comment:
You can use int for a unique identifier, but where do you set it? how do you make sure it is unique?

Reply:
fixed